### ISSUE TITLE
Invisibles at end of line not showing, cursor not moving

### ISSUE THREAD

**Issue by mquevill** [kevinsawicki/monokai#111]
I think this came up during an update to Atom, but I'm not sure if it's an issue with the Monokai theme or the language-c package, since it doesn't happen with every language. I'm thinking it might actually be related to braces ("{}"), since that seems to be when things get messed up.

My issue is that any invisible characters that I have set do not show at the end of the line. For instance, if I am typing on a line, it will not show me any spaces until I press a non-space character. In addition, it will not move my cursor! (See images below for demonstration.) I only have spaces and tabs shown for invisibles, but this happens with both of them.

Atom v1.56.0
Linux Mint 20.1

In a sample C++ file (not proper code):

Typed "Test case    " (4 spaces at end) with Monokai
![image](https://user-images.githubusercontent.com/17033543/114778825-5cc95480-9d43-11eb-9f86-e74907fbb06a.png)
Typed "Test case    1" (4 spaces) with Monokai [only typed "1" from previous]
![image](https://user-images.githubusercontent.com/17033543/114778855-65ba2600-9d43-11eb-92c7-42e60a23e898.png)
Typed "Test case    " (4 spaces at end) with Atom-Dark
![image](https://user-images.githubusercontent.com/17033543/114778772-4cb17500-9d43-11eb-9f15-c09499021b93.png)
Typed "Test case    1" (4 spaces) with Atom-Dark [only typed "1" from previous]
![image](https://user-images.githubusercontent.com/17033543/114778043-5d152000-9d42-11eb-895d-f7ee7e9f7dfa.png)

**Comment by jonyyz** [comment]
Invisibles don't show at all on 1.56 on Windows 10

Using Atom Dark:
![image](https://user-images.githubusercontent.com/41442131/114780154-03622500-9d45-11eb-99ee-17d262e2bced.png)

Using Monokai:
![image](https://user-images.githubusercontent.com/41442131/114780199-1117aa80-9d45-11eb-8b28-a23a8ceaa6f1.png)

**Comment by aaronclinger** [comment]
Trailing spaces also not working on macOS Big Sur and in a number of language packages (PHP, CSS, JavaScript, etc.)

I do believe it is an issue in combination with Atom v1.56.0. I rolled back to 1.55 and it seemed to work fine.

**Comment by jonyyz** [comment]
The issue is here in `index.less`:

![image](https://user-images.githubusercontent.com/41442131/114863819-43192300-9dbe-11eb-99e3-dd053d1ba799.png)

Remove that and the invisibles will show up again.  If you look at the theme template that ships with 1.56 compared to the theme template on github you'll notice there are some differences.  Apparently, that hack fix is no longer necessary?

**Comment by Tiross** [comment]
> The issue is here in `index.less`:
> 
> ![image](https://user-images.githubusercontent.com/41442131/114863819-43192300-9dbe-11eb-99e3-dd053d1ba799.png)
> 
> Remove that and the invisibles will show up again. 

Or add 
```less
.leading-whitespace,
.trailing-whitespace {
  display: inline;
}
```

into `~/.atom/styles.less`

**Comment by gustavoalfaro** [comment]
Fixed in #112
